django/otto/fixtures/apps.yaml

```diff
@@ -7,27 +7,31 @@
     user_group: AI assistant user
     features:
       - fields:
+          short_name: "qa"
```

Comment:
I was using these to more easily attribute Costs to particular Features.
However I think I would rather get rid of these, and not have a direct relationship between these Feature objects and Costs. This would be more flexible, e.g. for adding "librarian" feature which does not actually exist as a "Feature" in Otto.